networkx/algorithms/distance_measures.py

```diff
@@ -30,19 +30,22 @@ def extrema_bounding(G, compute="diameter"):
     compute : string denoting the requesting metric
        "diameter" for the maximal eccentricity value,
        "radius" for the minimal eccentricity value,
-       "periphery" for the set of nodes with eccentricity equal to the diameter
-       "center" for the set of nodes with eccentricity equal to the radius
+       "periphery" for the set of nodes with eccentricity equal to the diameter,
+       "center" for the set of nodes with eccentricity equal to the radius,
+       "eccentricities" for the maximum distance from each node to all other nodes in G
 
     Returns
     -------
     value : value of the requested metric
        int for "diameter" and "radius" or
-       list of nodes for "center" and "periphery"
+       list of nodes for "center" and "periphery" or
+       dictionary of eccentricity values keyed by node for "eccentricities"
 
     Raises
     ------
     NetworkXError
-        If the graph consists of multiple components
+        If the graph consists of multiple components or
+        If the compute parameter is passed an invalid argument
```

Comment:
```suggestion
    ValueError
        If `compute` is not one of "diameter", "radius", "periphery",
        "center", or "eccentricities".
```
Minor suggestion to make this more informative. See below for suggestion on changing the exception type.